jerry-ext/handle-scope/handle-scope.c

```diff
@@ -15,6 +15,9 @@
 
 #include <stdlib.h>
 #include "handle-scope-internal.h"
+#include "jext-common.h"
+
+JERRYX_STATIC_ASSERT (JERRYX_SCOPE_PRELIST_SIZE < 32, JERRYX_SCOPE_PRELIST_SIZE_must_be_less_than_size_of_uint8_t);
```

Comment:
These should be all capital letters.

Reply:
This was copy-pasted here from existing code. But updated now.